pkgs/by-name/wi/windowtolayer/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitLab,
+  python3Packages,
+  unstableGitUpdater,
+  rustfmt,
+}:
+rustPlatform.buildRustPackage {
+  pname = "windowtolayer";
+  version = "0-unstable-2025-01-26";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.freedesktop.org";
+    owner = "mstoeckl";
+    repo = "windowtolayer";
+    rev = "5ddb3a2834c834af4ec412bb2ba2c77431953d51";
+    hash = "sha256-PzKx8lkWDJGVj1Azc/vfbSPYV7x5+ZnCGMAAK4o207Y=";
+  };
+
+  cargoHash = "sha256-XuSlbBLWUkPsA0DYC4qemvAGyynkkuznA5FGBKRmNso=";
+  useFetchCargoVendor = true;
+
+  nativeBuildInputs = with python3Packages; [
+    python
+    rustfmt
+  ];
```

Why not just `python3`?

```suggestion
  nativeBuildInputs = [
    python3
    rustfmt
  ];
```

 I think at least `with` shouldn't be really used here

```suggestion
  nativeBuildInputs = [
    python3Packages.python
    rustfmt
  ];
```

---

I have changed it to `python3`.